modules/installation/pages/client-requirements.adoc

```diff
@@ -56,7 +56,7 @@ For non-{suse} operating systems, including {rhel}, {centos}, and {oracle}, only
 | {check}
 
 | {sles} for SAP
-| {x86_64}, {ppc}
+| {x86_64}, {ppc64le}
```

Comment:
Same comment as above: it does not make any sense without the version of SLES for SAP (12 or 15).